design/ceph/object/store.md

```diff
@@ -73,7 +74,124 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
   preservePoolsOnDelete: true
 ```
 
-If there is a `zone` section in object-store configuration, then the pool section in the ceph-object-zone resource will be used to define the pools.
+#### Pools shared by multiple CephObjectStore
+
+If user want to use existing pools for metadata and data, the pools must be created before the object store is created. This will be useful if multiple objectstore can share same pools. The detail of pools need to shared in `radosNamespaces` settings in object-store CRD. Now the object stores can consume same pool isolated with different namespaces. User can create via [Pool CRD](/Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md), this is need to present before the object store is created.
+
+```yaml
+  radosNamespaces:
+    metadataPoolName: rgw-meta-pool
+    dataPoolName: rgw-data-pool
+```
+
+To create the pools that will be shared by multiple object stores, create the following CephBlockPool CRs:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-meta-pool
+  spec:
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+ metadata:
+    name: rgw-data-pool
+  spec:
+    failureDomain: osd
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 2
+    application: rgw
+```
+
+The `.rgw.root` pool must be defined before the object store is created. This pool is used for storing the root info for all object stores.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-root
+  spec:
+    name: .rgw.root
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+```
+
+The pools for this configuration will be created as below:
+
+```bash
+# ceph osd pool ls
+.rgw.root
+rgw-meta-pool
+rgw-data-pool
+```
+
+And the pool configuration in zone is as below:
+
+```json
```

Comment:
This is really helpful, thank you!

Please make a note that the syntax here is `<pool-name>:<namespace>`. This will make it more clear for other readers that when namespacing is used, there are actually many, many namespaces created, especially in the metadata pool.

I would also suggest linking the RGW data layout doc, especially the compendium section in the design here as a reference: https://docs.ceph.com/en/latest/radosgw/layout/#appendix-compendium